### PR TITLE
exui-2469-hidden field should be removed from event submission

### PR DESCRIPTION
**Pull request by RiteshHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EXUI-2469

### Change description ###
hidden field should be removed from event submittion

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by MunishSharmaHMCTS — APPROVED** [review]

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Sonar failures

**Review by MunishSharmaHMCTS — DISMISSED** [review]

**Review by connorpgpmcelroy — DISMISSED** [review]
LGTM

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/services/valid-page-list-caseFields.service.ts [hunk 1]
```diff
@@ -27,11 +27,26 @@ export class ValidPageListCaseFieldsService {
     }
   }
 
-  public validPageListCaseFields(validPageList: WizardPage[], eventTriggerFields: CaseField[], formFields: object) : CaseField[] {
+  public validPageListCaseFields(validPageList: WizardPage[], eventTriggerFields: CaseField[], formFields: object, form?: any) : CaseField[] {
     const validPageListCaseFields: CaseField[] = [];
     validPageList.forEach(page => {
       if (this.isShown(page, eventTriggerFields, formFields)) {
-        page.case_fields.forEach(field => validPageListCaseFields.push(field));
+        page.case_fields.forEach(field => {
```
[the remark this answers is not in the document]
Comment (andywilkinshmcts) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/services/valid-page-list-caseFields.service.ts:34]: I did this refactor for you

**Review by andywilkinshmcts — APPROVED** [review]

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/services/valid-page-list-caseFields.service.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (RiteshHMCTS) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/services/valid-page-list-caseFields.service.ts:34]: thanks @andywilkinshmcts

**Review by andywilkinshmcts — APPROVED** [review]

**Pull request merged**